.github/actions/setup-go-test-env/action.yml

```
@@ -0,0 +1,94 @@
+            exit 1
+          fi
+        done
+
+    - name: Install migration tool and run migrations
+      working-directory: ${{ inputs.working-directory }}
+      shell: bash
+      env:
+        TEST_DATABASE_URL: "root:password@tcp(localhost:3306)/bocchi_test?parseTime=true&multiStatements=true"
+      run: |
+        go install -tags 'mysql' github.com/golang-migrate/migrate/v4/cmd/migrate@latest
+        migrate -path migrations -database "mysql://$TEST_DATABASE_URL" up
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)